packages/core/content-releases/admin/src/components/ReleaseListCell.tsx

```diff
@@ -0,0 +1,163 @@
+import * as React from 'react';
+
+import { Box, Flex, Popover, Typography } from '@strapi/design-system';
+import { Link } from '@strapi/design-system/v2';
+import { SortIcon } from '@strapi/helper-plugin';
+import { EntityService, Common } from '@strapi/types';
+import { useIntl } from 'react-intl';
+import { NavLink } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { useGetMappedEntriesInReleasesQuery } from '../services/release';
+import { useTypedSelector } from '../store/hooks';
+
+import type { CMAdminConfiguration, ListLayoutRow } from '@strapi/admin/strapi-admin';
+
+type Entity = EntityService.Result<Common.UID.Schema>;
+
+const Button = styled.button`
+  svg {
+    > g,
+    path {
+      fill: ${({ theme }) => theme.colors.neutral500};
+    }
+  }
+  &:hover {
+    svg {
+      > g,
+      path {
+        fill: ${({ theme }) => theme.colors.neutral600};
+      }
+    }
+  }
+  &:active {
+    svg {
+      > g,
+      path {
+        fill: ${({ theme }) => theme.colors.neutral400};
+      }
+    }
+  }
+`;
+
+const ActionWrapper = styled(Flex)`
+  svg {
+    height: ${4 / 16}rem;
+  }
+`;
+
+/* -------------------------------------------------------------------------------------------------
+ * useReleasesList
+ * -----------------------------------------------------------------------------------------------*/
+
+const useReleasesList = (entryId: string | number) => {
+  // @ts-expect-error TODO: fix Property 'uid' does not exist on type 'FormattedContentTypeLayout | null'
+  const { uid: contentTypeUid } = useTypedSelector(
```

Comment:
this might be `null` on initial load where the data has not populated the redux store no? Perhaps it's better to handle `null` below in the `useGetMappedEntriesInReleasesQuery` hook by skipping if it's not defined?